NAMESPACE

```diff
@@ -7,7 +7,6 @@ export(abline)
 export(arrows)
 export(axis)
 export(bgCol)
-export(bgCol.default)
```

Comment:
I deleted it because it seemed like a private function. Should we export those? I vote no but am up for doing what the group thinks is best.